crates/sui-graphql-rpc/src/extensions/logger.rs

```diff
@@ -141,29 +137,43 @@ impl Extension for LoggerExtension {
                         }
                     }
 
-                    error!(
-                        target: "async-graphql",
-                        "[Response] path={} message={}", path, err.message,
-                    );
-                } else {
-                    error!(
-                        target: "async-graphql",
-                        "[Response] message={}", err.message,
-                    );
+                    if let Some(async_graphql_value::ConstValue::String(val)) = error_code {
+                        if val.as_str() == code::INTERNAL_SERVER_ERROR {
+                            error!(
+                                query_id,
+                                session_id,
+                                error_code = val,
+                                "[Response] path={} message={}",
+                                path,
+                                err.message,
+                            );
+                        } else {
+                            info!(
+                                query_id,
+                                session_id,
+                                error_code = val,
+                                "[Response] path={} message={}",
+                                path,
+                                err.message,
+                            );
+                        }
+                    } else {
+                        warn!(
+                            query_id,
+                            session_id,
+                            error_code = code::UNKNOWN,
+                            "[Response] message={}",
+                            err.message,
+                        );
+                    }
                 }
             }
         } else if self.config.log_response {
             match operation_name {
                 Some("IntrospectionQuery") => {
-                    debug!(
-                        target: "async-graphql",
-                        "[Schema] {}: {}", self.session_id().await, resp.data
-                    );
+                    debug!(query_id, session_id, "[Response] {}", resp.data);
```

Comment:
nit: This one was tagged with `Schema` before -- was the change to `[Response]` before intentional?

Reply:
Oops, my mistake. Fixed!